test/Microsoft.ML.CodeAnalyzer.Tests/Code/ContractsCheckTest.cs

```diff
@@ -128,7 +128,7 @@ public async Task ContractsCheckFix()
                         VerifyCS.Diagnostic(ContractsCheckAnalyzer.NameofDiagnostic.Rule).WithLocation(25, 53).WithArguments("CheckUserArg", "name", "\"sp\""),
                         new DiagnosticResult("CS0122", DiagnosticSeverity.Error).WithLocation("Test1.cs", 752, 24).WithMessage("'ICancelable' is inaccessible due to its protection level"),
                         new DiagnosticResult("CS0122", DiagnosticSeverity.Error).WithLocation("Test1.cs", 752, 67).WithMessage("'ICancelable.IsCanceled' is inaccessible due to its protection level"),
-                        new DiagnosticResult("CS1503", DiagnosticSeverity.Error).WithLocation("Test1.cs", 753, 91).WithMessage("Argument 2: cannot convert from 'Microsoft.ML.Runtime.IHostEnvironment' to 'Microsoft.ML.Runtime.IExceptionContext'"),
+                        new DiagnosticResult("CS1503", DiagnosticSeverity.Error).WithLocation("Test1.cs", 753, 90).WithMessage("Argument 2: cannot convert from 'Microsoft.ML.Runtime.IHostEnvironment' to 'Microsoft.ML.Runtime.IExceptionContext'"),
```

Comment:
Just curious, why is this changing?

Reply:
I think it is because of the change here:

https://github.com/dotnet/machinelearning/blob/6dffa3280ef516c3607bc17c51844469848f2110/src/Microsoft.ML.Core/Utilities/Contracts.cs#L753

Here the misspelling *cancelled is corrected to "canceled," which has one fewer letter than its misspellings. Since this check is checking for the exact position of the error message (line 753, column 90), this correction brought this error message one character back.